openedx/core/djangolib/tests/utils.py

```diff
@@ -0,0 +1,61 @@
+from django.core.cache import caches
+from django.test import TestCase, override_settings
+
+
+class CacheSetupMixin(TestCase):
+    """
+    This class can be used to enable specific django caches for
+    specific the TestCase that it's mixed into.
+
+    Usage:
+
+    Use the ENABLED_CACHES to list the names of caches that should
+    be enabled in the context of this TestCase. These caches will
+    use a loc_mem_cache with the default settings.
+
+    Set the class variable CACHES to explicitly specify the cache settings
+    that should be overridden. This class will insert those values into
+    django.conf.settings, and will reset all named caches before each
+    test.
+
+
+    """
+
+    CACHES = None
+    ENABLED_CACHES = None
+
+    def setUp(self):
```

Comment:
Since these settings apply across the `TestCase`, shouldn't they be set in the `setUpClass` method, even if they're cleared at the `setUp` level?